doc/api/esm.md

```diff
@@ -120,12 +129,15 @@ given module specifier and parent file URL:
 const baseURL = new URL('file://');
 baseURL.pathname = process.cwd() + '/';
 
-export async function resolve(specifier,
-                              parentModuleURL = baseURL,
-                              defaultResolver) {
+export default function(parent) {
   return {
-    url: new URL(specifier, parentModuleURL).href,
-    format: 'esm'
+    async resolve(specifier,
+                  parentModuleURL = baseURL) {
+      return {
+        url: new URL(specifier, parentModuleURL).href,
+        format: 'esm'
+      };
+    }
```

Comment:
Perhaps make the example demonstrate chaining, by only applying to packages within the baseURL, while packages outside of the baseURL get the parent resolve or similar?

Reply:
fixed